code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1741,8 +1741,6 @@
 /mob/living/carbon/human/drop_from_inventory(var/obj/item/W, var/atom/target = null)
 	if(W in organs)
 		return
-	if(!W.canremove)
-		return
```

Comment:
The removal of that seems like it's going to cause issues with stuff that has this var set

Reply:
I added this check during the pr, but later removed because it was causing a lot of problem and runtimes. No augment uses canremove anymore.